NodeJS/src/RestClient.ts

```diff
@@ -130,7 +129,7 @@ export class RestClient {
     const id = await this.getSubAccount().then((response: { id: any }) => {
       return response.id
     })
-    console.log(`id is ${id}`)
+    logger.debug(`id is ${id}`)
```

Comment:
nit: for clarity, I'd recommend being more explicit on _what_ "id" this is
```suggestion
    logger.debug(`Sub Account App ID is: ${id}`)
```